calibimg.rst

```diff
@@ -4,9 +4,10 @@ CalibrateTask
 #############
 
 Given a properly characterized exposure (which means one with a PSF
-determined, commonly done previous to funning this Task by
-:doc:`CharacterizeImage <charimg>`), detect sources, measure their
-positions, and do a photometric measurement on them.
+determined and shipped along with the image, commonly done previous to
```

Comment:
I used the word 'shipped' vs. 'attached', seemed more accurate, but open to input on this.